requirements.txt

```
@@ -1,2 +1,69 @@
-openai==1.3.3
-gradio==4.4.1
+aiofiles==23.2.1 ; python_version >= "3.10" and python_version < "4.0"
```

requirements.txt文件更新了依赖列表，但没有更新poetry.lock文件。这可能导致依赖版本不一致的问题。

```suggestion
运行`poetry lock`命令更新poetry.lock文件，确保所有依赖版本一致。